src/components/investigators/InvestigatorForm.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import Swal from 'sweetalert2';
+import { Form, Button } from 'react-bootstrap';
+import { createInvestigator } from '../../redux/InvestigatorForm/investigatorFormSlice';
+import { fetchInvestigators } from '../../redux/investigators/investigatorSlice';
+
+const InvestigatorForm = () => {
```

Comment:
- [ ] When I try to add an investigator, this happens:

![image](https://user-images.githubusercontent.com/57421823/227433628-92739439-82e5-44ee-919a-e05fa0a2aaad.png)

The app is claiming I'm not logged in.

If this is a bug, kindly fix it so your app runs as intended. 👍